app/code/Magento/Customer/Controller/Section/Load.php

```diff
@@ -3,85 +3,83 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
+
 namespace Magento\Customer\Controller\Section;
 
-use Magento\Framework\App\Action\HttpGetActionInterface as HttpGetActionInterface;
-use Magento\Customer\CustomerData\Section\Identifier;
 use Magento\Customer\CustomerData\SectionPoolInterface;
-use Magento\Framework\App\Action\Context;
+use Magento\Framework\App\Action\HttpGetActionInterface as HttpGetActionInterface;
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\App\RequestInterface;
+use Magento\Framework\Controller\Result\Json as JsonResult;
 use Magento\Framework\Controller\Result\JsonFactory;
+use Magento\Framework\Escaper;
+use Zend\Http\AbstractMessage;
+use Zend\Http\Response;
 
 /**
- * Customer section controller
+ * Endpoint `customer/section/load` responsible for reloading sections of Customer's Local Storage
  */
-class Load extends \Magento\Framework\App\Action\Action implements HttpGetActionInterface
+class Load implements HttpGetActionInterface
 {
     /**
      * @var JsonFactory
      */
-    protected $resultJsonFactory;
-
-    /**
-     * @var Identifier
-     * @deprecated 100.2.0
-     */
-    protected $sectionIdentifier;
+    private $resultJsonFactory;
 
     /**
      * @var SectionPoolInterface
      */
-    protected $sectionPool;
+    private $sectionPool;
 
     /**
-     * @var \Magento\Framework\Escaper
+     * @var Escaper
      */
     private $escaper;
+    /**
+     * @var RequestInterface
+     */
+    private $request;
 
     /**
-     * @param Context $context
+     * @param RequestInterface $request
      * @param JsonFactory $resultJsonFactory
-     * @param Identifier $sectionIdentifier
      * @param SectionPoolInterface $sectionPool
      * @param Escaper $escaper
      */
     public function __construct(
-        Context $context,
+        RequestInterface $request,
         JsonFactory $resultJsonFactory,
-        Identifier $sectionIdentifier,
         SectionPoolInterface $sectionPool,
-        \Magento\Framework\Escaper $escaper = null
+        Escaper $escaper
     ) {
-        parent::__construct($context);
         $this->resultJsonFactory = $resultJsonFactory;
-        $this->sectionIdentifier = $sectionIdentifier;
         $this->sectionPool = $sectionPool;
-        $this->escaper = $escaper ?: $this->_objectManager->get(\Magento\Framework\Escaper::class);
+        $this->escaper = $escaper ?? ObjectManager::getInstance()->get(Escaper::class);
```

Comment:
We don’t need to have this fallback

Reply:
Aww... That is actually a huge mistake. Thank you!